eve/main.yml

```diff
@@ -363,6 +372,7 @@ stages:
             - lint
           haltOnFailure: true
       - SetPropertyFromCommand: *set_version_prop
+      - SetPropertyFromCommand: *set_short_version_prop
```

Comment:
Does not seem to be used.

Reply:
No indeed in premerge we not use it, but IMO having a property for short_version it's good anyway